src/components/List/helpers.tsx

```diff
@@ -18,7 +18,7 @@ export const Pollcolumns = () => {
       Filter: SelectColumnFilter,
       filter: 'includes',
       accessor: row => (timeLeft(row.endDate) === 'Ended' ? 'Ended' : 'Active'),
-      Cell: ({ row }) => timeLeft(row.original.endDate),
+      Cell: ({ row }) => (row.original.fetched ? timeLeft(row.original.endDate) : <Loading />),
```

Comment:
As discussed this morning, sometimes endDate comes in ms and some other in seconds, we use dates in seconds.
So it would be good to do something like `row.original.endDate >= 1e12 ? (row.original.endDate / 1e3).toFixed(0) : row.original.endDate` here or even better within `timeLeft` function